packages/webdriverio/src/commands/browser/react$.js

```diff
@@ -0,0 +1,39 @@
+/**
+ *
+ * Select React Composite Component or HTML Element based on passed selector
+ * API Reference: https://github.com/baruchvlz/resq#usage
+ *
+ * <example>
+    :react$.js
+    it('should get component', () => {
+        const browser = await remote({
+            // options
+        })
+        await browser.url('http://my-react-app.abc')
+
+        // Assuming you have a <MyComponent /> JSX element in your application
+        expect(await browser.react$('MyComponent')).toMatchObject({
+            node: null,
+            children: [],
+            state: {},
+            props: {},
+            isFragment: false
+        })
+    });
+ * </example>
+ *
+ * @alias browser.react$
+ * @param {String} Element to find in Virtual DOM
+ * @param {Object} Optional configuration for resq library
+ * @return {Object} ReactSelectorQueryNode provided by the resq library
+ * @type utility
+ *
+ */
+
+import { resq$, waitToLoadReact } from 'resq'
+
+export default async function react$(selector, options = {}) {
+    await waitToLoadReact(options.timeout, options.rootElement)
+
+    return resq$(selector)
```

Comment:
This won't work .. `resq` is code that runs in the browser and WebdriverIO runs on Node.js. We need to use the `execute` command here to inject resq and use its functionality.